app/views.py

```diff
@@ -89,11 +88,10 @@ def json_serial(obj):
     raise TypeError("Type %s not serializable" % type(obj))
 
 
-@csrf_exempt
 def add(request):
     ssn = request.POST['SSN']
     driver_license_number = request.POST['driver_license_number']
     pii_data = PiiData(social_security_number=ssn, driver_license_number=driver_license_number,
                        pub_date=datetime.now())
     pii_data.save()
-    return HttpResponse(str(pii_data.id))
+    return HttpResponseRedirect("/app/" + str(pii_data.id) + "/")
```

Comment:
was it a bug that you fixed

Reply:
I used ajax on UI for make request for this endpoint and in `onSuccess` method I changed the current page. For now, I do not need ajax request but I have to change page. So I use redirect for it.

Reply:
got it.